### PR TITLE
Added a few names and modified the order of one line

### PR DESCRIPTION
**Pull request by ArxdSilva**

### CODE REVIEW

File: names.json [hunk 1]
```diff
@@ -20,17 +21,19 @@
...
   "Cenoura",
   "Chibata",
   "Chulapa",
-  "Cobra caolha",
   "Cobra",
+  "Cobra caolha",
   "Cobra-cega",
   "Croquete",
   "Diglett",
+  "Dugtrio"
```
Comment (joaocunha) [inline names.json]: Missing comma.

**Comment by joaocunha** [comment]
Thanks for the contribution!

Please run `node test` before PRing to make sure the list is properly formatted (valid JSON, alphabetical, no duplicates, etc).

**Comment by ArxdSilva** [comment]
screen shot 2016-03-11 at 9 58 45 am
test passed :)

**Comment by joaocunha** [comment]
@ArxdSilva you might want to rebase with master. I pushed changes to the test suite yesterday.

**Comment by joaocunha** [comment]
@ArxdSilva sorry to be an :eggplant:, but you better branch off from master and submit the PR again. This PR as it is will mess up with the commit history.

Looking forward to it, we'll get to the 100th :eggplant: name :)

**Pull request closed without merging**